[Bug]: Unleash is Degraded (unhealthy) after first setup

### Error Message and Logs

2025-11-02T11:36:23.624733887Z Error: Cannot find module '/unleash/index.js'
2025-11-02T11:36:23.624736207Z     at Function._resolveFilename (node:internal/modules/cjs/loader:1401:15)
2025-11-02T11:36:23.624759767Z     at defaultResolveImpl (node:internal/modules/cjs/loader:1057:19)
2025-11-02T11:36:23.624762527Z     at resolveForCJSWithHooks (node:internal/modules/cjs/loader:1062:22)
2025-11-02T11:36:23.624765007Z     at Function._load (node:internal/modules/cjs/loader:1211:37)
2025-11-02T11:36:23.624767327Z     at TracingChannel.traceSync (node:diagnostics_channel:322:14)
2025-11-02T11:36:23.624769767Z     at wrapModuleLoad (node:internal/modules/cjs/loader:235:24)
2025-11-02T11:36:23.624772247Z     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:171:5)
2025-11-02T11:36:23.624774727Z     at node:internal/main/run_main_module:36:49 {
2025-11-02T11:36:23.624777087Z   code: 'MODULE_NOT_FOUND',
2025-11-02T11:36:23.624779367Z   requireStack: []
2025-11-02T11:36:23.624781648Z }
2025-11-02T11:36:23.624783808Z 
2025-11-02T11:36:23.624785968Z Node.js v22.17.1
2025-11-02T11:37:15.075292775Z node:internal/modules/cjs/loader:1404
2025-11-02T11:37:15.075324935Z   throw err;
2025-11-02T11:37:15.075328095Z   ^
2025-11-02T11:37:15.075330535Z 
2025-11-02T11:37:15.075349216Z Error: Cannot find module '/unleash/index.js'
2025-11-02T11:37:15.075352016Z     at Function._resolveFilename (node:internal/modules/cjs/loader:1401:15)
2025-11-02T11:37:15.075354576Z     at defaultResolveImpl (node:internal/modules/cjs/loader:1057:19)
2025-11-02T11:37:15.075369536Z     at resolveForCJSWithHooks (node:internal/modules/cjs/loader:1062:22)
2025-11-02T11:37:15.075371896Z     at Function._load (node:internal/modules/cjs/loader:1211:37)
2025-11-02T11:37:15.075374056Z     at TracingChannel.traceSync (node:diagnostics_channel:322:14)
2025-11-02T11:37:15.075375816Z     at wrapModuleLoad (node:internal/modules/cjs/loader:235:24)
2025-11-02T11:37:15.075377496Z     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:171:5)
2025-11-02T11:37:15.075379376Z     at node:internal/main/run_main_module:36:49 {
2025-11-02T11:37:15.075381096Z   code: 'MODULE_NOT_FOUND',
2025-11-02T11:37:15.075382696Z   requireStack: []
2025-11-02T11:37:15.075384376Z }
2025-11-02T11:37:15.075385856Z 
2025-11-02T11:37:15.075387416Z Node.js v22.17.1

### Steps to Reproduce

1. Create a new project using Unleash with Postgre
2. Change the domain of the unleash service
3. Deploy


### Example Repository URL

_No response_

### Coolify Version

v4.0.0-beta.434

### Are you using Coolify Cloud?

No (self-hosted)

### Operating System and Version (self-hosted)

Ubuntu 24.04.2 LTS

### Additional Information

_No response_

--- a/templates/compose/unleash-with-postgresql.yaml
+++ b/templates/compose/unleash-with-postgresql.yaml
@@ -20,9 +20,6 @@ services:
     depends_on:
       postgres:
         condition: service_healthy
-    command:
-      - node
-      - index.js
     healthcheck:
       test: "wget --no-verbose --tries=1 --spider http://127.0.0.1:4242/health || exit 1"
       interval: 1s

